03-Node-npm-modules/js/app.js

```diff
@@ -0,0 +1,30 @@
+requirejs.config({
+  baseUrl: 'js', // will automatically try to load modules from there, if successfully loaded, they will be accessible globally
+  paths: { // relative to the baseUrl
+  	jquery: '../node_modules/jquery/dist/jquery.min'
+  }
+});
+
+requirejs(['jquery', 'Movie', 'Director'], function ($, Movie, Director) {
+
+	var alien = new Movie('Alien');
+	var ridleyScott = new Director("Ridley Scott");
+	ridleyScott.set('quotes', ['Cast is everything.', 'Do what ...']);
+	alien.set('director', ridleyScott);
+	alien.get('director').speak();
+
+	$.each(alien.director.quotes, function(key, quotes) {
```

Comment:
Why not alien.get('director').get('quotes')? I mean this is good, but kind of makes your get/set functions useless if any property can be accessed directly as a property. That would make more sense if alien.director wasn't available, through some privacy.